tests/datasets.h

```diff
@@ -50,12 +50,12 @@ class LightDatasets {
 class HeavyDatasets {
 public:
     static inline const std::array<Dataset, 6> datasets_ = {
-            {{"adult.csv", 20873, ';', false},
-             {"CIPublicHighway.csv", 32696, ',', true},
-             {"EpicMeds.csv", 19617, '|', true},
-             {"EpicVitals.csv", 24082, '|', true},
-             {"iowa1kk.csv", 55285, ',', true},
-             {"LegacyPayors.csv", 34874, '|', true}}};
+            {{"adult.csv", 23075, ';', false},
+             {"CIPublicHighway.csv", 32696 /* TODO: update hash */, ',', true},
+             {"EpicMeds.csv", 50218, '|', true},
+             {"EpicVitals.csv", 24082 /* TODO: update hash */, '|', true},
+             {"iowa1kk.csv", 55285 /* TODO: update hash */, ',', true},
+             {"LegacyPayors.csv", 34874 /* TODO: update hash */, '|', true}}};
```

Comment:
Hashes are mismatched due to the additional quotes? Are we calculating hashes of a JSON string directly?

Reply:
Yes, `fletcher` invokes `GetFDsJSON`

Reply:
Hm, this doesn't seem right. Let's leave it as it is for now, however, I think we should calculate hash directly from left and right hand sides of an fd, not from any string representation of an fd (to avoid problems like this, we changed the representation and hashes are changed accordingly). @Mstrutov, what do you think? 

Anyways, could you fix this TODOs right away, Pyro/HyFD built in release mode will calculate it fast.

Reply:
TODOs fixed